mathics/format/text.py

```diff
@@ -31,7 +31,18 @@ def boxes_to_text(boxes, **options) -> str:
 
 
 def string(self, **options) -> str:
+    from mathics_scanner.characters import replace_wl_with_plain_text
+
     value = self.value
+    encoding = options.get("encoding", "Unicode")
+
+    # TODO: check if it makes sense to generalize this to
+    # other codepages...
+    if encoding in ("UTF8", "UTF-8"):
+        value = replace_wl_with_plain_text(value, True)
```

Comment:
Note that `replace_with_plain_text()` from mathics_scanner is deprecated. If we are using this there is something missing.

Reply:
@rocky, yes, I know. What I tried here was to have something that works with the current version of (1.2.4) mathics_scanner. Later I am going to try writing a version that works with both. If you have an idea about how to do it, I would appreciate the help.

Reply:
I have vague thoughts but I won't have time for a while to check or work things out for myself. 

I'll leaving to go hiking on Friday and will be back Monday.

Reply:
OK, there is no rush on solving this. I will continue with the other branches, which are more or less independent.